openfisca_core/entities.py

```diff
@@ -51,9 +51,21 @@ def check_role_validity(self, role):
         if role is not None and not type(role) == Role:
             raise Exception("{} is not a valid role".format(role))
 
-    def __call__(self, variable_name, period, options = [], **parameters):
+    def __call__(self, variable_name, period=None, options = [], **parameters):
```

Comment:
While PEP8 recommandes `x=y` in functions kwargs, we prefer `x = y` in OpenFisca for readability.